ENTITY/PlaySynchronizedMapEntityAnim.md

```diff
@@ -5,23 +5,23 @@ ns: ENTITY
 
 ```c
 // 0xB9C54555ED30FBC4 0xEB4CBA74
-BOOL PLAY_SYNCHRONIZED_MAP_ENTITY_ANIM(float p0, float p1, float p2, float p3, Any p4, Any p5, Any* p6, Any* p7, float p8, float p9, Any p10, float p11);
+BOOL PLAY_SYNCHRONIZED_MAP_ENTITY_ANIM(float x, float y, float z, float radius, int hash, int scene, char* animName, char* animDict, float blendInSpeed, float blendOutSpeed, bool, float);
```

Comment:
The last two parameters have no names, please create unique names for those parameters.